packages/flutter/lib/src/widgets/form.dart

```diff
@@ -465,6 +464,22 @@ class FormField<T> extends StatefulWidget {
   /// [FormState.save].
   final FormFieldSetter<T>? onSaved;
 
+  /// An optional property that forces the [FormFieldState] into an error state
+  /// by directly setting the [FormFieldState.errorText] property without
+  /// running the validator function.
+  ///
+  /// When the [forceErrorText] property is provided, the [FormFieldState.errorText]
+  /// will be set to the provided value, causing the form field to be considered
+  /// invalid and display the error message specified.
+  ///
+  /// The [forceErrorText] property is used to override the [InputDecoration.errorText]
+  /// value defined in the [InputDecoration] of the [TextFormField].
+  ///
+  /// Note: Using [forceErrorText] directly to set the error state bypasses the regular
```

Comment:
See the styleguide on the use of "Note": https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-empty-prose

Reply:
I'll remove the note.